x/oracle/keeper/slash_test.go

```diff
@@ -3,7 +3,7 @@ package keeper_test
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/staking"
-	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	stakingTypes "github.com/cosmos/cosmos-sdk/x/staking/types"
```

Comment:
this isn't the standard rn, we can change it eventually but just to keep consistency

```suggestion
	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
```

Reply:
https://github.com/ojo-network/ojo/pull/48/commits/cf3528a04c50ab8d497d41007de1f12841dabd4f